patches.json

```diff
@@ -101,5 +101,132 @@
         }
       }
     }
+  },
+  "MDVA-30195": {
+    "magento/magento2-base": {
+      "Fixes the issue where cron jobs fail if database name is too long, resulting in categories not being updated on the frontend.": {
+        ">=2.3.1 <=2.3.4-p2": {
+          "file": "os/MDVA-30195__fix_to_handle_long_database_names__2.3.4-p2.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30106": {
+    "magento/magento2-base": {
+      "Fixes an issue where during checkout payments are not loaded with “Cannot read property ‘length’ of null ” error in JS console.": {
+        ">=2.3.0 <=2.4.0": {
+          "file": "os/MDVA-30106__fix_validation_for_possible_null_objects_on_validation_rules__2.3.4.patch"
+        }
+      }
+    }
+  },
+  "MDVA-28656": {
+    "magento/magento2-base": {
+      "Fixes the issue where if an order was placed with no payment information required (for example, with 100% discount) and an invoice was created for the order, the order status changes to Closed instead of Complete.": {
+        ">=2.3.1 <=2.4.0": {
+          "file": "os/MDVA-28656__fixes_order_status_issue_when_invoicing_zero_payment_orders__2.3.5-p1.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30209": {
+    "magento/magento2-base": {
+      "Fixes the issue where the customer group was changed to default if customer updated their account information.": {
+        ">=2.3.0 <=2.3.3-p1": {
+          "file": "os/MDVA-30209__fixes_customer_group_id_during_account_update__2.3.0.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30123": {
+    "magento/magento2-base": {
+      "Fixes the issue where attribute option labels are not translated correctly for GraphQL queries.": {
+        ">=2.3.4 <=2.4.0": {
+          "file": "os/MDVA-30123__fixes_attribute_option_labels_translation_for_graphql_queries__2.3.4.patch"
+        }
+      }
+    }
+  },
+  "MDVA-29996": {
+    "magento/magento2-ee-base": {
+      "Fixes the issue where after enabling category permission, the category page is not getting cached by Full Page Cache.": {
+        ">=2.3.3 <=2.4.0": {
```

Comment:
The fix will be delivered in Magento 2.4.2
I propose to include 2.4.1 in version constraint

Reply:
fixed